docs/competition/event/incident-management.md

```diff
@@ -49,7 +49,18 @@ Due to the impact of stopping the competition, ability to do so is restricted to
 Stopping the competition will be indicated by a call over the radio of "This is _role_; stop the competition."
 This call should be repeated at least once, and every minute or so until acknowledged by one of the other aforementioned roles.
 
-At this point the following should occur;
+### Urgency call
+
+Any volunteer is able to raise an urgency call. This indicates that they feel a situation exists which may qualify as a major incident, and the roles able to stop the event should respond immediately.
+
+This should be indicated by a call over the radio of "Potential incident at _location_; emergency roles respond."
+This call should be repeated at least once, and every minute or so until acknowledged by one of the other aforementioned roles or the Head Shepherd.
+
+Should the Head Shepherd receive an Urgency call from a Shepherd, they should repeat this on the main radio network to ensure all.
```

Comment:
You seem to have stopped mid sentence.

Reply:
Argh, for some reason I only got shown this comment post-merge. Will fix

Reply:
PR #187 should account for this